spec/mailers/rrd_completed_mailer_spec.rb

```diff
@@ -16,6 +16,8 @@
 
     let(:bp_readings_count) { 1234 }
 
+    let(:active_medications) { 1234 }
```

Comment:
```suggestion
    let(:active_medications_count) { 1111 }
```
Also uses a different number so that tests that erroneously uses `bp_readings_count` will fail.